src/process-env.ts

```diff
@@ -5,13 +5,15 @@ const schema = z.object({
 	SUBMISSION_DISCORD_WEBHOOK_THREAD_ID: z.string(),
 })
 
-const result = schema.safeParse(import.meta.env)
-if (!result.success) {
```

Comment:
Instead of removing this wholesale maybe we move it behind a `import.meta.env.CONTEXT === 'production'` check? This would then only error for production builds (could maybe warn in other situations?)

Docs: https://docs.netlify.com/configure-builds/environment-variables/#read-only-variables

Reply:
Oh yeah that's interesting!  I try to avoid depending on Netlify's built-in variables to avoid issues when they don't exist in `astro dev`, but that actually makes sense in this case

Reply:
Turns out the fallback code gets a little wonky here checking `import.meta.env.CONTEXT` in addition to using `safeParse` to check for errors

Going to leave the easier fix here for now to unblock community PRs and open add it to the list of backlog tasks to revisit